pybossa/api/__init__.py

```diff
@@ -1022,3 +1023,55 @@ def get_project_progress(project_id=None, short_name=None):
         return Response(json.dumps(response), status=200, mimetype="application/json")
     else:
         return abort(403)
+
+
+@jsonpify
+@csrf.exempt
+@blueprint.route('/project/<int:project_id>/clone',  methods=['POST'])
+@blueprint.route('/project/<short_name>/clone',  methods=['POST'])
+@login_required
+def project_clone(project_id=None, short_name=None):
+
+    if current_user.is_anonymous:
+        return abort(401)
+    if short_name:
+        project = project_repo.get_by_shortname(short_name)
+    else:
+        project = project_repo.get(project_id)
+    if not project:
+        return abort(404)
+    if not (current_user.admin or (current_user.subadmin and current_user.id in project.owners_ids)):
+        return abort(403)
+
+    payload = json.loads(request.form['request_json']) if 'request_json' in request.form else request.json
```

Comment:
Consider removing the `form` data payload because this method is marked as `csrf.exempt`